tests/swarm/test_bandwidth_governor.py

```
@@ -0,0 +1,460 @@
+        peer = create_agent_id("SAT001")
+        
+        # Set utilization to 75%
+        governor.global_bucket._tokens = 500  # 25% available
+        
+        # NORMAL should be throttled
+        result = governor.acquire_tokens(peer, 100, MessagePriority.NORMAL)
+        assert result is False
+    
+    def test_critical_passes_at_100_percent(self):
+        """Test CRITICAL passes even at 100%."""
+        config, agent_id = create_config()
```

Ruff (0.14.10)

Unpacked variable `agent_id` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)